src/GfGiftAidField.php

```diff
@@ -33,10 +34,15 @@ public static function addSelectedPriceFieldSetting(int $position, int $form_id)
         if (25 !== $position || empty($form_id)) {
             return;
         }
+        $exists = GFAPI::form_id_exists($form_id);
+        if (empty($exists)) {
+            return;
+        }
```

Comment:
```suggestion
        if (! GFAPI::form_id_exists($form_id)) {
            return;
        }
```
stop doing empty checks on boolssssss!!!!!!